3_Risk.md

```diff
@@ -7,6 +7,7 @@ Qualified Committers | Contributions over time and what components they commit t
 User Dependency | Number of users who are aware that they depend on the software over time

```

Comment:
Not sure what this is saying/measuring.  Are we saying it's a bad thing for a code to be used by a lot of users?

Reply:
This came out of the OSSNA focus group. In the case of OpenSSH, few users knew that they depended on the software. In the group we talked about a potential action that may result from this information, for example communicating that the bug existed, giving it a name "heartbleed", and educating the public in layman's terms. If people do not know they depend on a software, they may not bother updating or reporting issues. - we are not saying anything about usage, only about awareness.